### PR TITLE
feat: Modal documentation

### PR DESCRIPTION
**Pull request by suevalov**
# Purpose of PR

Updated `Modal` documentation.

### CODE REVIEW

**Comment by domarku** [comment]
LGTM!

**Review by domarku — APPROVED** [review]

File: packages/components/modal/src/README.mdx [hunk 1]
```diff
@@ -5,213 +5,121 @@ status: 'stable'
...
+
+### Using ModalLauncher
+
+When you want to open a modal from a function (outside of React component code), use [ModalLauncher.](../components/modal-launcher)
+
+## Props (API reference)
+
+<Props of="Modal" storybookPath="/story/components-modal--basic" />
+
 ## Content recommendations
```
Comment (bgutsol) [inline packages/components/modal/src/README.mdx:78]:
```suggestion
## Content guidelines
```

**Review by bgutsol — COMMENTED** [review]
Is the readme for `ModalConfirm` also should have been migrated in this PR, or do we have a different PR/story for that one?

**Comment by suevalov** [comment]
@bgutsol `ModalConfirm` comes separately

**Pull request merged**